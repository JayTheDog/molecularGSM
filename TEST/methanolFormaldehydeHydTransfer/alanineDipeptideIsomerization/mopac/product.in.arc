

                     SUMMARY OF PM6 CALCULATION, Site No: 14762

                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 18:41:14 2016
                                                       No. of days left =   0

           Empirical Formula: C6 H12 N2 O2  =    22 atoms

PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =       -106.40354 KCAL/MOL =    -445.19241 KJ/MOL
          TOTAL ENERGY            =      -1841.44379 EV
          ELECTRONIC ENERGY       =      -9387.11218 EV
          CORE-CORE REPULSION     =       7545.66840 EV
          GRADIENT NORM           =          0.96100
          DIPOLE                  =          3.61046 DEBYE    POINT GROUP:       C1  
          NO. OF FILLED LEVELS    =         29
          IONIZATION POTENTIAL    =          9.626277 EV
          HOMO LUMO ENERGIES (EV) =         -9.626  0.727
          MOLECULAR WEIGHT        =        144.173
          COSMO AREA              =        184.70 SQUARE ANGSTROMS
          COSMO VOLUME            =        182.80 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    17    H    22     6.20440
            H    13    O    10     6.09727
            H    18    H    12     4.41259
          SCF CALCULATIONS        =         46
          WALL-CLOCK TIME         =          0.539 SECONDS
          COMPUTATION TIME        =          0.542 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  C     2.63104080 +1  -1.10600619 +1  -1.08725145 +1
  C     1.45543396 +1  -0.24740630 +1  -0.72035145 +1
  N     1.11133058 +1  -0.24504096 +1   0.63261737 +1
  C     0.01329696 +1   0.56685930 +1   1.20972899 +1
  C    -1.37125396 +1  -0.04719713 +1   0.85689640 +1
  N    -1.75827825 +1  -0.02735367 +1  -0.46554590 +1
  C    -3.04860173 +1  -0.60677657 +1  -0.88688973 +1
  O     0.82841174 +1   0.40424342 +1  -1.54604999 +1
  C     0.09349572 +1   2.03473062 +1   0.80707590 +1
  O    -2.07627213 +1  -0.49891347 +1   1.74950078 +1
  H     2.30929600 +1  -1.91283825 +1  -1.76618433 +1
  H     3.15469223 +1  -1.57159768 +1  -0.24639691 +1
  H     3.36755249 +1  -0.50947959 +1  -1.65054701 +1
  H     1.62293070 +1  -0.81977912 +1   1.29655651 +1
  H     0.10154464 +1   0.48802229 +1   2.33297757 +1
  H    -1.13184386 +1   0.31149759 +1  -1.20645397 +1
  H    -2.89639877 +1  -1.54509325 +1  -1.44639968 +1
  H    -3.59619998 +1   0.10594039 +1  -1.52533207 +1
  H    -3.68106694 +1  -0.83699737 +1  -0.00568018 +1
  H    -0.74352179 +1   2.60739952 +1   1.22576671 +1
  H     0.06811229 +1   2.16627001 +1  -0.28735871 +1
  H     1.02402391 +1   2.49624879 +1   1.15983004 +1
 
