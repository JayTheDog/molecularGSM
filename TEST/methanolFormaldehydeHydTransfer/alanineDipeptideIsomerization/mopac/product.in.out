 *******************************************************************************
 ** Site#: 14762        For non-commercial use only     Version 15.168L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.168L web: HTTP://OpenMOPAC.net         Days remaining:   0    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                              PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Fri Dec  2 18:41:14 2016  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *  NOSYM      - POINT-GROUP SYMMETRY SET TO C1
 *  T=         - A TIME OF     10.0 MINUTES REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          2.64378500  *  -1.12102800  *  -1.07399100  *
     2       C          1.44089000  *  -0.25765300  *  -0.73156200  *
     3       N          1.11354900  *  -0.19823800  *   0.58761800  *
     4       C         -0.00298000  *   0.56883900  *   1.17333300  *
     5       C         -1.39082500  *  -0.04839100  *   0.86304600  *
     6       N         -1.76843900  *  -0.01235800  *  -0.44119500  *
     7       C         -3.01569800  *  -0.60246400  *  -0.89301600  *
     8       O          0.82252900  *   0.34444500  *  -1.61350100  *
     9       C          0.08115500  *   2.06599200  *   0.84755600  *
    10       O         -2.08250200  *  -0.49232900  *   1.77525500  *
    11       H          2.32377100  *  -1.91487600  *  -1.75460200  *
    12       H          3.13164700  *  -1.56863500  *  -0.20441700  *
    13       H          3.36980200  *  -0.50540200  *  -1.61108800  *
    14       H          1.59399600  *  -0.82962800  *   1.21033700  *
    15       H          0.09465600  *   0.43960900  *   2.25302000  *
    16       H         -1.05927300  *   0.24857500  *  -1.12287900  *
    17       H         -2.84972600  *  -1.54682100  *  -1.42653800  *
    18       H         -3.54507000  *   0.08469000  *  -1.56094800  *
    19       H         -3.63181700  *  -0.80093700  *  -0.01550300  *
    20       H         -0.73490600  *   2.59396600  *   1.34994200  *
    21       H          0.01338700  *   2.24845300  *  -0.22477800  *
    22       H          1.02949900  *   2.47395100  *   1.21008000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        2.6438   -1.1210   -1.0740
     2         C        1.4409   -0.2577   -0.7316
     3         N        1.1135   -0.1982    0.5876
     4         C       -0.0030    0.5688    1.1733
     5         C       -1.3908   -0.0484    0.8630
     6         N       -1.7684   -0.0124   -0.4412
     7         C       -3.0157   -0.6025   -0.8930
     8         O        0.8225    0.3444   -1.6135
     9         C        0.0812    2.0660    0.8476
    10         O       -2.0825   -0.4923    1.7753
    11         H        2.3238   -1.9149   -1.7546
    12         H        3.1316   -1.5686   -0.2044
    13         H        3.3698   -0.5054   -1.6111
    14         H        1.5940   -0.8296    1.2103
    15         H        0.0947    0.4396    2.2530
    16         H       -1.0593    0.2486   -1.1229
    17         H       -2.8497   -1.5468   -1.4265
    18         H       -3.5451    0.0847   -1.5609
    19         H       -3.6318   -0.8009   -0.0155
    20         H       -0.7349    2.5940    1.3499
    21         H        0.0134    2.2485   -0.2248
    22         H        1.0295    2.4740    1.2101

 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: C6 H12 N2 O2  =    22 atoms



      MOLECULAR POINT GROUP   :   C1  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =   29

      Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:    80.371 HEAT: -101.9184
 CYCLE:     2 TIME:   0.031 TIME LEFT: 10.00M  GRAD.:    64.103 HEAT: -103.4069
 CYCLE:     3 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:    40.400 HEAT: -104.5535
 CYCLE:     4 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    61.789 HEAT: -104.5860
 CYCLE:     5 TIME:   0.023 TIME LEFT: 10.00M  GRAD.:    28.670 HEAT: -105.3857
 CYCLE:     6 TIME:   0.023 TIME LEFT: 10.00M  GRAD.:    28.563 HEAT: -105.5689
 CYCLE:     7 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:    13.962 HEAT: -105.7143
 CYCLE:     8 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:    11.327 HEAT: -105.8241
 CYCLE:     9 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    10.487 HEAT: -105.9629
 CYCLE:    10 TIME:   0.023 TIME LEFT: 10.00M  GRAD.:    15.657 HEAT: -106.0297
 CYCLE:    11 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    18.156 HEAT: -106.0671
 CYCLE:    12 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:    17.708 HEAT: -106.0992
 CYCLE:    13 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    11.165 HEAT: -106.1696
 CYCLE:    14 TIME:   0.023 TIME LEFT: 10.00M  GRAD.:     7.843 HEAT: -106.2070
 CYCLE:    15 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.832 HEAT: -106.2351
 CYCLE:    16 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:     8.700 HEAT: -106.2572
 CYCLE:    17 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:     9.017 HEAT: -106.2767
 CYCLE:    18 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     5.516 HEAT: -106.3147
 CYCLE:    19 TIME:   0.031 TIME LEFT:  9.99M  GRAD.:     3.004 HEAT: -106.3260
 CYCLE:    20 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     3.036 HEAT: -106.3338
 CYCLE:    21 TIME:   0.016 TIME LEFT:  9.99M  GRAD.:     3.390 HEAT: -106.3505
 CYCLE:    22 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     2.628 HEAT: -106.3597
 CYCLE:    23 TIME:   0.016 TIME LEFT:  9.99M  GRAD.:     2.035 HEAT: -106.3670
 CYCLE:    24 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     2.255 HEAT: -106.3719
 CYCLE:    25 TIME:   0.016 TIME LEFT:  9.99M  GRAD.:     2.116 HEAT: -106.3744
 CYCLE:    26 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     1.647 HEAT: -106.3777
 CYCLE:    27 TIME:   0.016 TIME LEFT:  9.99M  GRAD.:     1.429 HEAT: -106.3800
 CYCLE:    28 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     1.188 HEAT: -106.3811
 CYCLE:    29 TIME:   0.016 TIME LEFT:  9.99M  GRAD.:     1.146 HEAT: -106.3812
 CYCLE:    30 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     1.421 HEAT: -106.3817
 CYCLE:    31 TIME:   0.016 TIME LEFT:  9.99M  GRAD.:     1.281 HEAT: -106.3828
 CYCLE:    32 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     1.109 HEAT: -106.3859
 CYCLE:    33 TIME:   0.016 TIME LEFT:  9.99M  GRAD.:     1.199 HEAT: -106.3915
 CYCLE:    34 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     1.161 HEAT: -106.3951
 CYCLE:    35 TIME:   0.008 TIME LEFT:  9.99M  GRAD.:     1.007 HEAT: -106.4001
 CYCLE:    36 TIME:   0.016 TIME LEFT:  9.99M  GRAD.:     0.961 HEAT: -106.4035

     GRADIENT =  0.96100  IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 18:41:14 2016
                                                       No. of days left =   0




          FINAL HEAT OF FORMATION =       -106.40354 KCAL/MOL =    -445.19241 KJ/MOL


          TOTAL ENERGY            =      -1841.44379 EV
          ELECTRONIC ENERGY       =      -9387.11218 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =       7545.66840 EV
          COSMO AREA              =        184.70 SQUARE ANGSTROMS
          COSMO VOLUME            =        182.80 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.96100
          IONIZATION POTENTIAL    =          9.626277 EV
          HOMO LUMO ENERGIES (EV) =         -9.626  0.727
          NO. OF FILLED LEVELS    =         29
          MOLECULAR WEIGHT        =        144.173

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    17    H    22     6.20440
            H    13    O    10     6.09727
            H    18    H    12     4.41259


          SCF CALCULATIONS        =         46
 
          WALL-CLOCK TIME         =          0.539 SECONDS
          COMPUTATION TIME        =          0.541 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          2.63104080  *  -1.10600619  *  -1.08725145  *
     2       C          1.45543396  *  -0.24740630  *  -0.72035145  *
     3       N          1.11133058  *  -0.24504096  *   0.63261737  *
     4       C          0.01329696  *   0.56685930  *   1.20972899  *
     5       C         -1.37125396  *  -0.04719713  *   0.85689640  *
     6       N         -1.75827825  *  -0.02735367  *  -0.46554590  *
     7       C         -3.04860173  *  -0.60677657  *  -0.88688973  *
     8       O          0.82841174  *   0.40424342  *  -1.54604999  *
     9       C          0.09349572  *   2.03473062  *   0.80707590  *
    10       O         -2.07627213  *  -0.49891347  *   1.74950078  *
    11       H          2.30929600  *  -1.91283825  *  -1.76618433  *
    12       H          3.15469223  *  -1.57159768  *  -0.24639691  *
    13       H          3.36755249  *  -0.50947959  *  -1.65054701  *
    14       H          1.62293070  *  -0.81977912  *   1.29655651  *
    15       H          0.10154464  *   0.48802229  *   2.33297757  *
    16       H         -1.13184386  *   0.31149759  *  -1.20645397  *
    17       H         -2.89639877  *  -1.54509325  *  -1.44639968  *
    18       H         -3.59619998  *   0.10594039  *  -1.52533207  *
    19       H         -3.68106694  *  -0.83699737  *  -0.00568018  *
    20       H         -0.74352179  *   2.60739952  *   1.22576671  *
    21       H          0.06811229  *   2.16627001  *  -0.28735871  *
    22       H          1.02402391  *   2.49624879  *   1.15983004  *

                             CARTESIAN COORDINATES

   1    C        2.631040800    -1.106006186    -1.087251446
   2    C        1.455433960    -0.247406296    -0.720351447
   3    N        1.111330581    -0.245040965     0.632617367
   4    C        0.013296964     0.566859303     1.209728990
   5    C       -1.371253959    -0.047197126     0.856896402
   6    N       -1.758278255    -0.027353669    -0.465545904
   7    C       -3.048601731    -0.606776565    -0.886889731
   8    O        0.828411740     0.404243419    -1.546049991
   9    C        0.093495718     2.034730623     0.807075902
  10    O       -2.076272130    -0.498913473     1.749500783
  11    H        2.309296004    -1.912838255    -1.766184327
  12    H        3.154692233    -1.571597680    -0.246396909
  13    H        3.367552489    -0.509479586    -1.650547015
  14    H        1.622930701    -0.819779117     1.296556514
  15    H        0.101544644     0.488022293     2.332977571
  16    H       -1.131843859     0.311497590    -1.206453969
  17    H       -2.896398772    -1.545093253    -1.446399685
  18    H       -3.596199978     0.105940393    -1.525332072
  19    H       -3.681066939    -0.836997366    -0.005680182
  20    H       -0.743521789     2.607399517     1.225766714
  21    H        0.068112285     2.166270013    -0.287358711
  22    H        1.024023911     2.496248786     1.159830035


           Empirical Formula: C6 H12 N2 O2  =    22 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -33.11400 -32.12176 -30.88734 -30.19402 -27.91075 -26.15608 -24.78314 -22.93815
 -21.22368 -19.91856 -18.33067 -18.01117 -16.49193 -15.98928 -15.77522 -15.35650
 -14.72932 -14.61003 -14.35325 -14.11613 -13.61594 -13.42822 -13.34896 -13.12725
 -12.68246 -11.00770 -10.82867  -9.85650  -9.62628   0.72702   0.89979   1.62670
   1.84046   3.02473   3.19161   3.57379   3.81475   4.37621   4.53568   4.78863
   4.88428   5.05583   5.19275   5.43817   5.50725   5.53432   5.61340   5.78646
   5.99084   6.15253   6.52145   6.86709


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C          -0.614069        4.6141     1.10088     3.51319
    2          C           0.657114        3.3429     1.06300     2.27988
    3          N          -0.524348        5.5243     1.50516     4.01919
    4          C           0.024565        3.9754     1.10758     2.86786
    5          C           0.526288        3.4737     1.09023     2.38349
    6          N          -0.447114        5.4471     1.48877     3.95835
    7          C          -0.284481        4.2845     1.08515     3.19933
    8          O          -0.583579        6.5836     1.85424     4.72934
    9          C          -0.509412        4.5094     1.08932     3.42009
   10          O          -0.570245        6.5702     1.85589     4.71435
   11          H           0.203507        0.7965     0.79649
   12          H           0.181908        0.8181     0.81809
   13          H           0.204830        0.7952     0.79517
   14          H           0.277040        0.7230     0.72296
   15          H           0.159884        0.8401     0.84012
   16          H           0.304382        0.6956     0.69562
   17          H           0.143832        0.8562     0.85617
   18          H           0.146339        0.8537     0.85366
   19          H           0.178280        0.8217     0.82172
   20          H           0.170031        0.8300     0.82997
   21          H           0.189720        0.8103     0.81028
   22          H           0.165526        0.8345     0.83447
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     3.145    -0.438    -0.800     3.275
 HYBRID         0.302    -0.179    -0.080     0.360
 SUM            3.447    -0.617    -0.880     3.610


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.10088   1.13283   1.17581   1.20456
    2  C   1.06300   0.78047   0.74099   0.75842
    3  N   1.50516   1.37959   1.48104   1.15856
    4  C   1.10758   0.88599   0.92640   1.05547
    5  C   1.09023   0.84374   0.75279   0.78696
    6  N   1.48877   1.21744   1.59331   1.14760
    7  C   1.08515   0.93919   1.10923   1.15092
    8  O   1.85424   1.61261   1.53629   1.58044
    9  C   1.08932   1.18816   1.02669   1.20524
   10  O   1.85589   1.62253   1.55456   1.53727
   11  H   0.79649
   12  H   0.81809
   13  H   0.79517
   14  H   0.72296
   15  H   0.84012
   16  H   0.69562
   17  H   0.85617
   18  H   0.85366
   19  H   0.82172
   20  H   0.82997
   21  H   0.81028
   22  H   0.83447

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.54 SECONDS

 == MOPAC DONE ==
